AppCenterDataStorage/AppCenterDataStorageTests/MSDBDocumentStoreTests.m

```diff
@@ -16,14 +27,153 @@ @implementation MSDBDocumentStoreTests
 
 - (void)setUp {
   [super setUp];
+  [MSUtility deleteItemForPathComponent:kMSDBDocumentFileName];
   self.sut = [MSDBDocumentStore new];
-  self.dbStorageMock = OCMClassMock([MSDBStorage class]);
-  self.sut.dbStorage = self.dbStorageMock;
 }
 
 - (void)tearDown {
+  [MSUtility deleteItemForPathComponent:kMSDBDocumentFileName];
   [super tearDown];
-  [self.dbStorageMock stopMocking];
+}
+
+- (void)testReadUserDocumentFromLocalDatabase {
+
+  // If
+  NSString *documentId = @"12829";
+  NSString *partitionKey = @"user";
+  NSString *accountId = @"dabe069b-ee80-4ca6-8657-9128a4600958";
+  NSString *eTag = @"398";
```

Comment:
Those variables are used multiple times in this test. They can be refactored to constants.